kolibri/plugins/coach/assets/src/views/plan/assignments/IndividualLearnerSelector.vue

```diff
@@ -347,6 +355,11 @@
     font-size: 16px;
   }
 
+  .table-data {
+    padding-top: 6px;
```

Comment:
Visually, this looks good, but I feel like the checkbox in the first column is forcing us to add more vertical padding than we need. So hopefully in the future, we can figure out how to get the checkbox to take up less vertical space and then remove this `padding-top` rule.